newIDE/app/src/SceneEditor/index.js

```diff
@@ -836,6 +896,10 @@ export default class SceneEditor extends React.Component<Props, State> {
     if (this._objectsList) this._objectsList.forceUpdateList();
   };
 
+  forceUpdateLayersList = () => {
+    if (this._layersList) this._layersList.onLayerModified();
```

Comment:
I would recommend avoiding "hijacking" `onLayerModified` because: 
- it's called "onXXX" so usually seen as a method that the component will call when things have been modified "inside" the component rather than outside.
- it's triggering potentially other stuff than just force updating.

Instead, just call `forceUpdate` on layersList. The name is fine, and it's totally understandable that because we modified the layers list outside, we now must force the update of the layers editor.